test/libsolidity/syntaxTests/abiEncoder/select_v1_quoted_string.sol

```diff
@@ -0,0 +1 @@
+pragma abicoder "v1";
```

Comment:
Is something like this valid?
```solidity
pragma experimental abicoder "v1";
```

Reply:
No. There is a test for multiple tokens after experimental already.